libraries/AP_HAL_ChibiOS/RCOutput.cpp

```diff
@@ -119,6 +119,10 @@ void RCOutput::init()
     hal.gpio->pinMode(57, 1);
 #endif
 
+#ifdef HAL_WITH_BIDIR_DSHOT
```

Comment:
Shouldn't this one check HAL_WITH_ESC_TELEM as well?

Reply:
I think we are going to tie ourselves in knots if we try and make HAL_WITH_ESC_TELEM optional

Reply:
No, not tie in knots. I successfully did it with #19 : 
- It compiles fine on *normal* builds
- It compiles fine on *bdshot* builds
- It compiles fine with ESC device drivers active.
- It compiles fine with ESC device drivers de-active.

Reply:
Fixed a different way